airbyte-workers/src/main/java/io/airbyte/workers/DefaultReplicationWorker.java

```diff
@@ -302,7 +304,14 @@ private static Runnable getReplicationRunnable(final AirbyteSource source,
               try {
                 recordSchemaValidator.validateSchema(messageOptional.get().getRecord());
               } catch (final RecordSchemaValidationException e) {
-                LOGGER.warn(e.getMessage());
+                if (validationErrors.size() < 10) {
```

Comment:
I'm hoping logging errors for 10 streams will be sufficient

Reply:
This will log 10 different validation messages, not necessarily 10 different streams.

It probably does make sense to put a cap on the number of errors that we log to avoid filling up the logs, just in case there are thousands of unique schema validation error messages. However, I would raise the cap here to 100, since 10 feels low to me. 

Also, with the way that this conditional is written, we will stop incrementing the count of validation errors once there are 10 unique errors in the map. This conditional on the size should be moved into the `if (exceptionCount == null)` block below, to prevent us from adding new values to the map once we have hit the cap.

Reply:
ah yes 10 different streams, assuming that each stream tends to throw the same validation error (which is what we've seen)

Reply:
as far as the count - are you saying there should be a separate variable to track the total number of validation errors? right now it's just the count of each particular error, but stopping adding new errors after 10 (can up this to 100)